### ISSUE TITLE
Migrate all stylus styles to sass

### ISSUE THREAD

**Issue by MrNaif2018** [bitcartcc/bitcart-store#316]
We currently have at least 2 style engines utilized: sass and stylus. There is no need in stylus as we have sass already. We need to migrate all styles to sass to be able to customize theme colors in the future
Some examples:
https://github.com/bitcartcc/bitcart-store/blob/ca1ba4e8d782999c9c02e46fbddf9ec806dd6bfb/components/Switch.vue#L24-L26
https://github.com/bitcartcc/bitcart-store/blob/8909d56c7c2cf431ae93e8b97958a418cd3aa366/pages/index.vue#L53-L68
https://github.com/bitcartcc/bitcart-store/blob/8909d56c7c2cf431ae93e8b97958a418cd3aa366/components/Sidebar.vue#L64-L101

Also any improvements to the existing files are welcome!

**Comment by MaximeKoitsalu** [comment]
Hey, I could take a look at this issue if it's alright with you?

**Comment by MrNaif2018** [comment]
Sure! Thank you very much

**Comment by MrNaif2018** [comment]
@MaximeKoitsalu Did you have any problems with setting it up?
It requires our merchants api instance running (from main bitcart repository).
You have two ways:
1. Use our demo api instance to just test UI changes
```
export BITCART_STORE_API_URL=https://api.bitcartcc.com
yarn dev
```
2. Run merchants api locally (I do it this way, but I often make changes in both projects, for UI changes it is not required): see https://github.com/bitcartcc/bitcart/blob/master/CONTRIBUTING.md#setting-up-python-development-environment

Let me know if it was not very obvious/undocumented. Also I guess a better error message should be displayed when merchants api is not accessible. Will open a separate issue for that

**Comment by MaximeKoitsalu** [comment]
Thanks for the guide, I haven't gotten to it yet but will get to it tomorrow 🙏

**Comment by MrNaif2018** [comment]
Fixed by #321
